src/components/storefront/ProductDetails/ProductDetails.js

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { Heading, Flex, Image, Text, Button } from '@chakra-ui/react';
+import AdditionalContent from './AdditionalContent';
+
+const ProductDetails = ({ product }) => {
+  const { images, title, description, variants } = product;
+  const productSku = variants && variants[0].sku;
+  const productImage = images && images[0].src;
+  const productPrice = variants && variants[0].price;
+
+  const handleAddToShoppingBag = () => {
+    alert('To be implemented');
+  };
+
+  return (
+    <Flex py={20} px={32}>
```

Comment:
we don't need it to be perfect, but this is also looking pretty wonky on small screens: 

![image](https://user-images.githubusercontent.com/26701004/112728377-5fcbe480-8efd-11eb-972b-ad409df6ae1d.png)


Again, don't go ham spending too much time, but some suggestions: 
* less margin on small screens
* maybe a single column layout on small screens.

btw also I also noticed the app is completely messed up on phone layouts, so maybe this will be a lot of work and we can have another ticket as a "responsive design pass" later. lmk your thoughts, feel free to timebox/push responsive design to separate ticket